lnrpc/rpc.proto

```diff
@@ -290,72 +559,136 @@ message HTLC {
 }
 
 message ActiveChannel {
+    /// Whether this channel is active or not
     bool active = 1 [json_name = "active"];
+    /// The identity pubkey of the remote node
     string remote_pubkey = 2 [json_name = "remote_pubkey"];
+
+    /**
+    ChannelPoint is the outpoint (txid:index) of the funding transaction. With this value, Bob will be able to generate a signature for Alice's version of the commitment transaction.
+    */
     string channel_point = 3 [json_name = "channel_point"];
+
+    /**
+    ChannelID is the unique channel ID for the channel. The first 3
+    bytes are the block height, the next 3 the index within the block,
+    and the last 2 bytes are the output index for the channel.
+    */
     uint64 chan_id = 4 [json_name = "chan_id"];
 
+    /// The total amount of funds held in this channel
     int64 capacity = 5 [json_name = "capacity"];
+
+    /// This node's current balance in this channel
     int64 local_balance = 6 [json_name = "local_balance"];
+
+    /// The counterparty's current balance in this channel
     int64 remote_balance = 7 [json_name = "remote_balance"];
 
+    /**
+    CommitFee is the amount calculated to be paid in fees for the current set of commitment transactions. The fee amount is persisted with the channel in order to allow the fee amount to be removed and recalculated with each channel state update, including updates that happen after a system restart.
```

Comment:
Previously, were not including the name of the field in the comment, but the following set of comments do include it. Seems inconsistent?